Updates on some of the repo files (#129)
BlackPythonDevs/blackpythondevs: .github/ISSUE_TEMPLATE/task.md

---
name: New Task
label: task
assignees: '@.rai (CoderBlee)'
about: 'Create a new task to be tracked'
---

# Task Title: BlackPythonDevs Hackathon & Learning Fest
## Date & Time:
- **Date:** [Choose a Date]
- **Time:** [Choose a Time] (e.g., 10:00 AM – 4:00 PM)
## Venue:
- Location: [Choose a Venue or Virtual Platform]
- Address: [Physical Address if in-person, or platform details if virtual]

## Tasks - Do these in Order

### Objective:

> To bring together Python, hackerthors and absolute beginners enthusiasts to collaborate on open-source projects, learn new skills, and network with like-minded individuals.

Reviewer: ```suggestion
To bring Python enthusiasts, hackathon participants, and absolute beginners in collaborative open-source projects, fostering skill development and creating opportunities for networking with like-minded individuals.
```

Reviewer: Rewording to bring your point home.

Author: Thank you soo much for looking into it. I was discouraged and honestly did not know what to do anymore. I also learned not to contribute in bulk

**6. Hackathon Begins (2 hours)**
   - **Time:** 1:00 PM – 3:00 PM
   - **Details:** Teams work on their projects. Provide mentors or facilitators to assist with technical guidance and support.

**7. Show & Tell (1 hour)**
   - **Time:** 3:00 PM – 4:00 PM
   - **Details:** Each team presents their work. Focus on what was achieved, challenges faced, and next steps. This could be done in a fun, creative way (e.g., live demos, short skits). Very important have a content creator for more exposure and growth of the meetup. Also for growth of the campaign and get more ideas from viewers

**8. Closing Remarks & Awards (30 mins)**
   - **Time:** 4:00 PM – 4:30 PM
   - **Details:** Summarize the day’s achievements, thank all participants and volunteers. Award small prizes for categories like “Most Innovative Project,” “Best Collaboration,” etc.

**9. After-Party (Optional)**
   - **Time:** 4:30 PM – 6:00 PM
   - **Details:** Continue networking and relaxing. Offer refreshments and perhaps a casual discussion or Q&A session with the keynote speaker.

### Team Activities:

1. **Mini Workshops:**

>    - **“Advanced Python Techniques”** – A workshop focusing on more complex aspects of Python. If possible also sell the BPD merch

Reviewer: ```suggestion
   - **“Advanced Python Techniques”** – A workshop focusing on more complex aspects of Python. If possible also sell the BPD merchandise.
```

   - **“Open Source Best Practices”** – Tips and strategies for contributing to open-source projects.

2. **Creative Coding Challenges:**
   - Short coding challenges or puzzles related to the day’s projects. Offer small rewards for those who solve them.

3. **Mentorship Circles:**

>    - Form small groups where experienced developers mentor beginners. This fosters knowledge sharing and builds community.

Reviewer: ```suggestion
   - Form small groups where experienced developers mentor beginners. This fosters knowledge sharing and builds the community.
```

### Promotion:

1. **Social Media:**

>    - Create event pages on platforms like Twitter, LinkedIn, and Facebook. Use hashtags relevant to Python and open-source communities.

Reviewer: ```suggestion
   - Create event pages on platforms like Twitter( X), LinkedIn, and Facebook. Use hashtags relevant to Python and open-source communities.
```
